python/benchmark/test_gen_data.py

```diff
@@ -244,15 +244,12 @@ def test_make_regression(
     "density_curve, shuffle",
     [
         ("None", "True"),
-        ("Linear", "True"),
         ("Linear", "False"),
         ("Exponential", "False"),
         pytest.param("Exponential", "True", marks=pytest.mark.slow),
     ],
 )
-@pytest.mark.parametrize(
-    "n_chunks", ["10", pytest.param("100", marks=pytest.mark.slow)]
-)
+@pytest.mark.parametrize("n_chunks", ["100"])
```

Comment:
Is this case not actually slow?   Same question for other change above.

Reply:
Was trying to reduce the number of tests to avoid too large testing runtime. A number greater than 10 would good since 10 is the default and we want to test for something more rigorous. Do you have any good number in mind (like 20 maybe?)